File: BEE 1045.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var a = parseFloat(line.shift())
var b = parseFloat(line.shift())
var c = parseFloat(line.shift())
var arr = [a, b, c];
arr.sort(function(a, b){return a - b});
a = arr[2];
b = arr[1];
c = arr[0];

if (a >= (b + c)){
    console.log("NAO FORMA TRIANGULO");
}
else if ((a * a) == ((b * b) + (c * c))){
    console.log("TRIANGULO RETANGULO");
}
else if ((a * a) > ((b * b) + (c * c))){
    console.log("TRIANGULO OBTUSANGULO");
}
else if ((a * a) < ((b * b) + (c * c))){
    console.log("TRIANGULO ACUTANGULO");
}

if (a == b && a == c){
    console.log("TRIANGULO EQUILATERO");
}

if ((a == b && a != c) || (b == c && b != a) || (a == c && a != b)){
    console.log("TRIANGULO ISOSCELES");
}

File: BEE 1045.py
a, b, c = map(float, input().split(" "))
list_sorted = [a,b,c]
list_sorted.sort(reverse=True)
A = list_sorted[0]
B = list_sorted[1]
C = list_sorted[2]
if A>=(B+C):
    print("NAO FORMA TRIANGULO")
elif (A*A)==(B*B+C*C):
    print("TRIANGULO RETANGULO")
elif (A*A)>(B*B+C*C):
    print("TRIANGULO OBTUSANGULO")
elif (A*A)<(B*B+C*C):
    print("TRIANGULO ACUTANGULO")
if A==B and A==C:
    print("TRIANGULO EQUILATERO")
if (A==B and A!=C) or (A==C and A!=B) or (B==C and B!=A):
    print("TRIANGULO ISOSCELES")